### PR TITLE
Disable APZ for source editors. Fixes #129.

### LINKED ISSUE #129: Disable APZ for source editors
The current version of APZ (async pan zoom) can cause tearing effects in the source editor where the line numbers float around disconnected from the code.

See https://bugzilla.mozilla.org/attachment.cgi?id=8722367

### PR DESCRIPTION
**Pull request by gabrielluong**
Fixes #129.

r? @jlongster

### CODE REVIEW

**Comment by jasonLaster** [comment]
👍

**Comment by gabrielluong** [comment]
Cancel the r? for now until I pass the eslint

**Comment by jasonLaster** [comment]
@gabrielluong just fixed the eslint bug. it was a bug with a lint rule we added this morning and hadn't adjusted that pending PR.

File: public/js/components/Editor.js [hunk 1]
```diff
@@ -87,6 +87,29 @@ const Editor = React.createClass({
     });
 
     this.editor.on("gutterClick", this.onGutterClick);
+
+    // Disable APZ for source editors. It currently causes the line numbers to
+    // "tear off" and swim around on top of the content. Bug 1160601 tracks
+    // finding a solution that allows APZ to work with CodeMirror.
+    this.editor.getScrollerElement().addEventListener("wheel", ev => {
```
Comment (jasonLaster) [inline public/js/components/Editor.js]: I think the onWheel logic should land in `util/editor.js` as it's a utility for the editor exclusively.

`this.editor.getScrollerElement().addEventListener("wheel", ev =>  onWheel(this.editor, ev);`

Comment (jlongster) [inline public/js/components/Editor.js]: I agree, let's keep the component clean and free of hacks. It's unfortunate that we have to do this.... but thanks a bunch for fixing it! I was noticing this and was wondering what was going on.

If you move this code to `util/editor.js` we'll merge this.

**Comment by jasonLaster** [comment]
Looks great

**Pull request merged**